src/test/java/ai/vespa/cloud/docsearch/VespaDocProductionTest.java

```diff
@@ -11,30 +11,28 @@
 import java.net.http.HttpResponse;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @ProductionTest
 public class VespaDocProductionTest {
 
-    HttpClient httpClient = HttpClient.newBuilder()
-            .version(HttpClient.Version.HTTP_2)
-            .build();
-
     @Test
     void verifyMetrics() throws IOException, InterruptedException {
-
         // Can use publicly available resources only, so use open query interface
         // Here, ensure > 50 documents about ranking
         HttpRequest req = HttpRequest.newBuilder()
-                .GET()
-                .uri(URI.create("https://doc-search.vespa.oath.cloud/search/?query=ranking&ranking=documentation&locale=en-US&hits=1"))
-                .build();
-        HttpResponse<String> res = httpClient.send(req, HttpResponse.BodyHandlers.ofString());
+                                     .GET()
+                                     .uri(URI.create("https://doc-search.vespa.oath.cloud/search/?query=ranking&ranking=documentation&locale=en-US&hits=1"))
```

Comment:
as this is a sample app, an YQL query would be better, but this does not change it for the worse :-)

Reply:
Hehe. We have YQL in the other tests, though. Prodduction tests must be ddifferent, at least until we can provide them with read access.